pkg/cmd/start/start.go

```diff
@@ -19,8 +22,12 @@ import (
 
 const serviceCACertPath = "/var/run/configmaps/service-ca-bundle/service-ca.crt"
 
+type operatorController interface {
+	Run(ctx context.Context, controller *controllercmd.ControllerContext) error
+}
```

Comment:
I think we don't need this interface. It can be `func runOperator(operator controller.Operator, cfg *controllercmd.ControllerCommandConfig)` or am I missing anything?

Reply:
Just a leftover from a failed attempt :)